spec/features/questions/index_spec.rb

```diff
@@ -0,0 +1,15 @@
+require "rails_helper"
+require 'pry'
+RSpec.describe "display all questions" do
+  it "display all question" do
+    RatingQuestion.create!(title: "Is drinking water good?")
```

Comment:
Move this setup to `let!` blocks at the top of the test.

Reply:
is this to improve the visual experience?

Reply:
Yes that's right. Setup should be outside of the expectation.